src/main/java/chess/ChessBoard.java

```diff
@@ -29,9 +41,12 @@ public ChessBoard() {
     public Piece[][] getBoard() {
         return this.board;
     }
-
+    /**method to move a piece**/
+    public void move2(final Position p, final Piece piece) {
+        board[p.getColumn()][p.getRow()] = piece;
+    }
```

Comment:
This method does the same action that addPiece does.

Reply:
Change made can you review it?